recipes/tkwant/bld.bat

```diff
@@ -1 +1 @@
-%PYTHON% setup.py install --single-version-externally-managed --record=record.txt
+%PYTHON% setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This file shouldn't be touched, but I was seeing this issue locally as well so :shrug:

Reply:
yeah I can get rid of it... `git reset --hard` has no effect.